Taskfile.yml

```diff
@@ -9,21 +9,44 @@ vars:
   DOCKERCMD_SERVER: 'itkdev-docker-compose-server'
 
 tasks:
-  dist:clone:
-    summary: Clone freescout distribution.
+  dist:build:
+    summary: Install Freescout distribution
     cmds:
-      - git clone https://github.com/freescout-help-desk/freescout {{ .TARGET_DIR }}
+      - sh -c "wget https://github.com/freescout-help-desk/freescout/archive/refs/tags/{{.VERSION}}.tar.gz"
+      - sh -c "mkdir freescout"
+      - sh -c "tar -xzvf ./{{.VERSION}}.tar.gz"
+      - sh -c "mv freescout-{{.VERSION}}/* freescout"
```

Comment:
This will not move any .dot files.

You should use -C on the tar command: `tar -xzf ./{{.VERSION}}.tar.gz -C freescout `

Reply:
Also no need to run verbose on the tar command

Reply:
That will create a freescout/freescout-1.8.145/* structure where i will have to move files around anyways. I changed the tar command to remove verbose and change the mv command to move full directory instead of dir/*

Reply:
the add `--strip-components 1` which will remove the first level when un-packing

Reply:
Added.